src/ui/screens/Council/CandidateScreen.tsx

```diff
@@ -168,3 +176,15 @@ function Voter({accountId}: {accountId: AccountId}) {
     />
   );
 }
+
+const styles = StyleSheet.create({
+  twitter: {
```

Comment:
Both twitter and matrix are just links. You can use the primary color from the `useTheme` hook instead of hardcoding.

Also please check the app in both light and dark mode. Since the colors we get from `useTheme` are coming from the theme, it is recommended to use instead of hardcoding.